### DISCUSSION TITLE
Is the code for "Go to definition" "Find All References" open source?

### DISCUSSION THREAD

**Discussion by strentler** [microsoft/vscode-cpptools#7710] [Q&A]
Hi,

I wonder if this is open source and can be adjusted to my needs?

**Comment by michelleangela** [comment]
@strentler 
The code for any feature (like "Go to Definition" or "Find All References") that requires tag parsing or IntelliSense is not open source. The only open source portion of the C++ extension is the Typescript component available here on the repository.

**Reply by michelleangela** [reply]
If there is a feature that is not supported by C++ extension and if this feature could benefit a lot of users, please feel free to submit a "feature request" issue at https://github.com/microsoft/vscode-cpptools/issues/new/choose.
